tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -1310,6 +1310,77 @@ TEST_F(MatmulSchedulerTest, EpilogueAlpha) {
   NVF_CHECK(outputs[0].allclose(tref, 0.001, 0.001));
 }
 
+TEST_F(MatmulSchedulerTest, MatmulHopperCastBiasCast) {
```

Comment:
I think this is fine for now but since this test is not Hopper-specific iin the future (once #3440 is trimmed and merged) we could probably remove it and just unguard some other tests like the MatmulOp and LinearOp translation tests which use the same fusion but with a `matmul` instead of `fusedMultiplySum`.